docs/software/programming.md

```diff
@@ -3,7 +3,8 @@
 ## Overview
 
 - [**Python - high-level interpreted programming language**](#python)
-- [**Docker - Build, ship, and run distributed applications**](#docker)  
+- [**Docker - Build, ship, and run distributed applications**](#docker)
+- [**Docker-Compose - run multi-container applications on Docker**](#docker-compose)
```

Comment:
Also here the wish to change the upper/lower case.

Reply:
done